### ISSUE TITLE
PS4 shuts off when running exploit

### ISSUE THREAD

**Issue by erwangse** [ChendoChap/pOOBs4#27]
Hi. I followed every step closely and my PS4 shuts off by itself when running the exploit.

Here's what happens :
- Format USB with correct exfathax image from the repo (master branch)
- Load the exploit in the PS4 browser (Karo host)
- Wait for the JS alert to show up (I don't click OK)
- Plug in USB stick into the PS4
- Wait for 30 seconds approx. (No notification pops up)
- PS4 cuts off completely, like cutting off power instantly.

I believe I flashed the image correctly, first with DD, which gave me the same result as win32diskimager later on.

I'm running ver. 9.00 on a PS4 slim and everything is ideal I believe.

**Comment by OneNYBoi** [comment]
I'm having the same issue atm, except I'm using a usb to sd card adapter, I thought that was the issue but after seeing someone else with the same issue I guess not.

**Comment by xXS4ALLx** [comment]
> I'm having the same issue atm, except I'm using a usb to sd card adapter, I thought that was the issue but after seeing someone else with the same issue I guess not.

Just because someone else is having same issue as you doesn't mean your not the only one.
Why not just get a proper usb flash drive and stop using half assed usb to sd adaptor to see if it will resolve your issue. Reading on other people's issue thinking its related to you aint going to help.
Get productive and start troubleshooting and eliminating what might be causing your issue, as mentioned above, start by using a usb flash drive.

**Comment by mrdude2478** [comment]
For the people with this issue, try using a different usb pen drive, or reflash the usb img - I've noticed that removing the usb drive by just pulling it out can corrupt it - reflashing the img seems to fix the crashing/freezing issues. I've had to reflash mine two or thee times now each times it's flashed it seems to fix those freezing issues.

**Comment by erwangse** [comment]
Can confirm with @mrdude2478 
I've fixed this issue by flashing the img with rufus, as it gets exclusive write access so Windows can't mount it at the end of the flash.
Moreover, I've had to use a different USB pen drive, as I originally used those compact flash-only sticks without a controller on it.

TL;DR: Try this new combination :
- Use a ***different*** flash drive model and form.
- Flash with rufus
- Download a fresh img

I confirm this fixed the issue for me.

*It would be nice to have a MD5 checksum of the img in the repo, just to check we're good.*
